pkgs/development/python-modules/pillow/default.nix

```diff
@@ -31,6 +31,9 @@ buildPythonPackage rec {
     ++ stdenv.lib.optionals (isPyPy) [ tk libX11 ];
 
   # NOTE: we use LCMS_ROOT as WEBP root since there is not other setting for webp.
+  # NOTE: we patch the `disable_platform_guessing` setting here, instead of
+  # passing the `--disable-platform-guessing` command-line option, since the
+  # command-line option doesn't work when we run tests.
```

Comment:
Can you also summarize here briefly why, this has to be patched? Basically what you wrote in the commit message.

Reply:
@Mic92 - Done; amended and force-pushed the commit.